### ISSUE TITLE
one-context-reconfigure-delayed failed with result 'dependency' on Centos7

### ISSUE THREAD

**Issue by jsfrerot** [OpenNebula/addon-context-linux#119]
Hi, I get the following error after upgrading from one-context 5.4.2.1 to 5.6.0 Using onebula 5.6. I have no issues when using one-context 5.4.2.1.

/var/log/messages shows: 
`Aug 27 14:04:16 customvm systemd: Job one-context-reconfigure-delayed.service/start failed with result 'dependency'.`

```
[root@customvm ~]# systemctl status one-context-reconfigure-delayed
● one-context-reconfigure-delayed.service - OpenNebula delayed reconfiguration script
   Loaded: loaded (/usr/lib/systemd/system/one-context-reconfigure-delayed.service; disabled; vendor preset: disabled)
   Active: inactive (dead)

Aug 27 14:04:16 customvm systemd[1]: Dependency failed for OpenNebula delayed reconfiguration script.
Aug 27 14:04:16 customvm systemd[1]: Job one-context-reconfigure-delayed.service/start failed with result 'dependency'.
```

**Comment by vholer** [comment]
(Delayed) reconfiguration is triggered on NIC attach or CD-ROM change to do the live reconfiguration, but it may be triggered also during the boot (when NICs are cold plugged). The dependency error is just a safety check that the "cold plug" triggered reconfiguration isn't done sooner than the regular contextualization done on boot. It usually isn't a real error.

Please check the `one-context-local` and `one-context`, these are the main services responsible for contextualization, they should be enabled and finish successfully.

**Comment by jsfrerot** [comment]
Looks like the services are disabled when installing the package.
Is this expected ?

```
[root@customvm ~]# systemctl status one-context
● one-context.service - OpenNebula contextualization script
   Loaded: loaded (/usr/lib/systemd/system/one-context.service; disabled; vendor preset: disabled)
   Active: inactive (dead)
[root@customvm ~]# systemctl status one-context-local
● one-context-local.service - OpenNebula pre-networking contextualization script
   Loaded: loaded (/usr/lib/systemd/system/one-context-local.service; disabled; vendor preset: disabled)
   Active: inactive (dead)
```

**Comment by vholer** [comment]
Yes, it's not correct and you have to enable these services manually (or don't do package upgrade but clean installation).

There was a problem when the package cleanup scripts disabled these services not only on (old) package removal, but also on upgrade. Fixed since 5.6.0 in https://github.com/OpenNebula/addon-context-linux/issues/106.

**Comment by jsfrerot** [comment]
Ok, removing the package and re-installing it worked for me.
Thanks
